requirements/base.txt

```diff
@@ -10,3 +11,4 @@ python-dateutil==2.4.2
 pytz==2015.4
 six>=1.9.0
 tablib==0.10.0
+git+https://github.com/rbarrois/python-ldap@py3  # py3 branch, not on pypi
```

Comment:
move it (and django-auth-ldap) to prod-ldap.txt (and include prod.txt in it, where you'll include base.txt and put for example gunicorn here)